### PR TITLE
feat(DEV-8621): add discovery link to header

### PR DESCRIPTION
**Pull request by doubleppereira**
Ticket ID
[DEV-8621](https://app.clickup.com/t/2645698/DEV-8621)
[DEV-8646](https://app.clickup.com/t/2645698/DEV-8646)

Description
Add discovery link to header
Changed navbar brand onClick to point to up.cloud

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-8621 Add discovery link in navbar](https://app.clickup.com/t/2645698/DEV-8621)

File: components/AppNavbar.vue [hunk 1]
```diff
@@ -27,6 +27,14 @@ const handleNavigateSend = () => {
   }
 }
 
+const handleNavigationDiscovery = () => {
+  try {
+    window.open(BASE_UP_CLOUD_URL, '_self')
+  } catch (error) {
+    console.error(error)
+  }
```
Comment (dzbo) [inline components/AppNavbar.vue:35]: Is this try/catch useful here? I think there is no possibility that `window.open` throw an error.

File: components/AppNavbar.vue [hunk 2]
```diff
@@ -63,6 +71,14 @@ const browserSupportExtension = extensionStore.url !== ''
     @on-brand-click="handleBrandClick"
   >
     <div class="w-full flex items-center justify-end" slot="desktop">
+      <lukso-button
+        v-if="status.isConnected"
```
Comment (dzbo) [inline components/AppNavbar.vue:75]: I think we can remove `v-if` here, this link should be always visible.

Comment (doubleppereira) [reply components/AppNavbar.vue:75]: yes make sense. 👍🏼

**Review by richtera — CHANGES_REQUESTED** [review]
handleBrandClick already does the navigation. Please re-use that method.

**Comment by doubleppereira** [comment]
> handleBrandClick already does the navigation. Please re-use that method.

Actually is not the same navigation. handleBrandClick navigates to 'https://my.universalprofile.cloud/' and here we want to navigate to 'https://universalprofile.cloud/' but now that I am checking there is a overlap of this task with https://app.clickup.com/t/2645698/DEV-8646. I can change here to just point to https://universalprofile.cloud/ on both `Discovery` Nav button and header logo

**Comment by doubleppereira** [comment]
@richtera can you check again I closed https://github.com/lukso-network/wallet.universalprofile.cloud/pull/108 since it was overlapping the changes made here. Now it uses a single handle and both point to `up.cloud`

**Pull request merged**